### PR TITLE
Bump repo versions to 6.8

### PR DESCRIPTION
**Pull request by lhellebr**
There are still old versions of some repos - most notably Tools - being used.

Note that there were some even older versions than 6.7, e.g. Capsule 6.6 - I don't know why so I might break something fixing this, but I think it's more likely that someone just forgot to bump them before.

Tried one test that uses `rhst7`:

```
$ pytest tests/foreman/api/test_errata.py::ErrataTestCase::test_positive_install_in_hc
================================================ test session starts ================================================
[...]
===================================== 1 passed, 2 warnings in 555.63s (0:09:15) =====================================
```

### CODE REVIEW

File: robottelo/constants/__init__.py [hunk 1]
```diff
@@ -480,8 +490,8 @@
         'key': 'rhva6',
     },
     'rhva65': {
-        'name': ('Red Hat Enterprise Virtualization Agents for RHEL 6 Server RPMs x86_64 6.5'),
-        'version': '6.5',
+        'name': ('Red Hat Enterprise Virtualization Agents for RHEL 6 Server RPMs x86_64 6.8'),
+        'version': '6.8',
```
Comment (jameerpathan111) [inline robottelo/constants/__init__.py:494]: @lhellebr is this right? I am guessing, `rhva65` is referring to rhel version here and not satellite tools. Need to check.

File: robottelo/constants/__init__.py [hunk 2]
```diff
@@ -539,10 +549,10 @@
         'rhva6',
         'rhva65',
         'repo_name',
-        'Red Hat Enterprise Virtualization Agents for RHEL 6 Server RPMs x86_64 6.5',
+        'Red Hat Enterprise Virtualization Agents for RHEL 6 Server RPMs x86_64 6.8',
```
Comment (jameerpathan111) [inline robottelo/constants/__init__.py:552]: probably here as well. we should check.

File: robottelo/constants/__init__.py (hunk 1 above)
Comment (mirekdlugosz) [reply robottelo/constants/__init__.py:494]: I think Jameer is right, this refers to RHEL version, not Satellite. Newest RHEL 6 is 6.10.

Also, variable name `rhva65` should probably be changed as well (along with everything that might use this name...).

Comment (lhellebr) [reply robottelo/constants/__init__.py]: Yes, this refers to RHEL version. Restored 6.5 and added entries for 6.10.

**Review by jameerpathan111 — APPROVED** [review]
LGTM

**Review by mshriver — APPROVED** [review]

**Pull request merged**